src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -394,5 +394,108 @@ public static void EnumAsSByteMax()
             SimpleTestClass obj = JsonSerializer.Parse<SimpleTestClass>(s_jsonSByteEnum);
             Assert.Equal(SampleSByteEnum.Max, obj.MySByteEnum);
         }
+
+        [Fact]
+        public static void ToString_MaxValuePlus1_QuotedVsNotQuoted_QuotedThrows()
```

Comment:
The test name doesn't match then actual behavior (we don't throw). We also don't have quoted here.